content/doc/developer/publishing/releasing-experimental-updates.adoc

```diff
@@ -2,9 +2,16 @@
 title: Publishing Experimental Plugin Releases
 layout: developer
 ---
+link:https://www.jenkins.io/doc/book/managing/plugins/[The Plugin] Manager downloads plugins and their dependencies into a folder which enables easier import into an instance of Jenkins which benefits in avoiding the recreation of docker files or other implications of plugin management in Jenkins.
+This also provides more information about the plugins being downloaded.link:https://github.com/jenkinsci/plugin-installation-manager-tool[Plugin installation manager tool for Jenkins]. 
```

Comment:
```suggestion
link:https://github.com/jenkinsci/plugin-installation-manager-tool[The Plugin Manager tool] downloads plugins and their dependencies into a folder, which enables easier import into an instance of Jenkins.
This avoids the recreation of docker files, or other implications of plugin management, in Jenkins.
This also provides more information about the plugins being downloaded, such as available updates or security warnings.
```
updated to include the direct link in the first sentence, added some info about what information is provided, sentence per line formatting